project 1/styles.css

```diff
@@ -0,0 +1,70 @@
+body {
+    background-color: #DCE6D5;
```

Comment:
Consider also adding a `color` and `font-family` declaration to your `body` element -- it would save you from repeating the `font-family` rule especially 👍🏻